tests/functional/settings.js

```diff
@@ -172,27 +174,21 @@ define([
 
     'sign in, open settings in a second tab, sign out': function () {
       var windowName = 'sign-out inter-tab functional test';
-      var self = this;
-      return FunctionalHelpers.fillOutSignIn(this, email, FIRST_PASSWORD)
-        .then(function () {
-          return FunctionalHelpers.openSettingsInNewTab(self, windowName);
```

Comment:
This is where the original problem was, we didn't wait for the original tab to sign in before opening settings in the new tab.